src/components/createStore/addLogo.js

```diff
@@ -69,7 +69,7 @@ const AddLogo = props => {
 
   useEffect(() => {
     dispatch(creators.setLoading(false))
-  }, [])
+  }, [dispatch])
```

Comment:
@shaunorpen You sure this doesn't break anything?

Reply:
I ran all tests after the update and it passes. No new warnings. Can't think why it would break anything, but am prepared to learn!